_partials/_graphing-ohlcv-data.md

```diff
@@ -0,0 +1,33 @@
+## Graph OHLCV data
+
+When you have extracted the raw OHLCV data, you can use it to graph the result
+in a candlestick chart, using Grafana. To do this, you need to have Grafana set
+up to connect to your TimescaleDB database. For more information about how to do
+this, see the [Grafana setup instructions][grafana-setup].
+
+<Procedure>
+
+### Graphing OHLCV data
+
+1.  Ensure you have Grafana installed, and you are using the TimescaleDB
+    database that contains the Twelve Data stocks dataset set up as a
+    data source. For more information about how to do this, see the [Grafana
+    setup instructions][grafana-setup].
+1.  In Grafana, from the `Dashboards` menu, click `New Dashboard`. In the
+    `New Dashboard` page, click `Add a new panel`.
+1.  In the `Visualizations` menu in the top right corner, select `Candlestick`
+    from the list. Ensure you have set the Twelve Data stocks dataset as
+    your data source.
+1.  Click `Edit SQL` and paste in the query you used to get the OHLCV values.
+1.  In the `Format as` section, select `Table`.
```

Comment:
Might be good to explain why first in the paragraph. I'm not sure what this does.

Reply:
No concepts in the task topics. This is a How to content and we explain only how to and not why in tasks topics. In the concept topics, we explain the why.